packages/itwinui-react/src/core/ColorPicker/ColorInputPanel.tsx

```diff
@@ -23,6 +30,22 @@ type ColorInputPanelProps = {
    * @default ['hsl', 'rgb', 'hex']
    */
   allowedColorFormats?: ('hsl' | 'rgb' | 'hex')[];
+  /**
+   * Props for color input panel box.
+   */
+  panelLabelProps?: React.ComponentProps<'div'>;
+  /**
+   * Props for color input field container box.
+   */
+  inputContainerProps?: React.ComponentProps<'div'>;
+  /**
+   * Props for color input field box.
```

Comment:
can you try to go through all of these jsdoc comments to make them more descriptive and accurate? some of them look too vague or even wrong.

Reply:
I have reworded the JSDoc comments.